### PR TITLE
[WIP] Add default settings for facetting in demo

### PR DESCRIPTION
**Pull request by hparfr**
Add some facettings settings in demo 

[EDIT]: This PR is in WIP, because there is an ongoing refactoring by @florian-dacosta.

### CODE REVIEW

**Review by mourad-ehm — APPROVED** [review]

File: shopinvader_algolia/demo/backend_demo.xml [hunk 1]
```diff
@@ -7,18 +7,52 @@
 
 <record id="index_1" model="se.index">
     <field name="backend_id" ref="connector_algolia.backend_1_se_backend"/>
-    <field name="name">algolia-product</field>
+    <field name="name">demo_site_product_en_US</field>
```
Comment (bealdav) [inline shopinvader_algolia/demo/backend_demo.xml]: could be broken with https://github.com/akretion/connector-search-engine/pull/34

Comment (florian-dacosta) [reply shopinvader_algolia/demo/backend_demo.xml]: Yes, the field name is becoming computed and should not be overriden.

**Comment by lmignon** [comment]
@hparfr can you check the latests lint errors PLZ...

**Comment by hparfr** [comment]
Linting errors are not related to this module.

**Comment by florian-dacosta** [comment]
Anyway, after some discussions with @hparfr and @sebastienbeau, I think this PR is not needed anymore
@hparfr You confirm?
I have to make a module to automatically export the right facettings to algolia.
I should be able to do it soon, at the end of the week I hope.

**Comment by lmignon** [comment]
closing as suggested by https://github.com/akretion/odoo-shopinvader/pull/250#issuecomment-453973284

**Pull request closed without merging**